### ISSUE TITLE
Wrong line number output from swiftformat lint

### ISSUE THREAD

**Issue by dwirandytlvk** [nicklockwood/SwiftFormat#1310]
Hallo

I would like to ask about this issue that happend to me

When i run swiftformat --lint it give wrong line number, since line number 125 is empty line

<img width="1008" alt="Screen Shot 2022-11-16 at 16 18 47" src="https://user-images.githubusercontent.com/93510180/202140003-ef987efd-079a-4cac-b63c-36098280ca90.png">

### SwiftFormat output
```
/ExperienceEasyReservationPaxPreferenceView.swift:125:1: warning: (wrap) Wrap lines that exceed the specified maximum width.
```

After i run swiftformat auto correct, it seems swiftformat added space to line 125

<img width="1008" alt="Screen Shot 2022-11-16 at 16 24 03" src="https://user-images.githubusercontent.com/93510180/202141259-a994591b-f294-46fc-8bb6-ad949484a7c4.png">

Is it expected result?

### Configuration File
.swiftformat file
```
# options
--swiftversion 5.6
--self remove # redundantSelf
--indent 4 #indent
--ifdef no-indent #indent
--indentstrings true #indent
--wraparguments before-first # wrapArguments
--wrapparameters before-first # wrapArguments
--wrapcollections before-first # wrapArguments 
--wrapconditions after-first # wrapArguments
--wrapreturntype preserve #wrapArguments
--closingparen balanced # wrapArguments
--wraptypealiases before-first # wrapArguments
--funcattributes prev-line # wrapAttributes
--typeattributes prev-line # wrapAttributes
--wrapternary before-operators # wrap
--extensionacl on-extension # extensionAccessControl
--patternlet inline # hoistPatternLet
--typeblanklines preserve # blankLinesAtStartOfScope, blankLinesAtEndOfScope
--emptybraces spaced # emptyBraces
--allman false # braces
--trimwhitespace nonblank-lines

# We recommend a max width is 120
--maxwidth 200 # wrap

# rules
--rules braces
--rules blankLinesBetweenScopes
--rules consecutiveSpaces
--rules extensionAccessControl
--rules hoistPatternLet
--rules indent
--rules redundantParens
--rules redundantSelf
--rules redundantPattern
--rules redundantGet
--rules redundantFileprivate
--rules redundantRawValues
--rules strongifiedSelf
--rules typeSugar
--rules wrap
--rules wrapArguments
--rules wrapAttributes
--rules redundantInit
--rules redundantVoidReturnType
--rules spaceInsideBrackets
--rules spaceInsideBraces
--rules spaceAroundBraces
--rules spaceInsideParens
--rules spaceAroundParens
--rules blockComments
--rules spaceAroundComments
--rules spaceInsideComments
--rules blankLinesAtStartOfScope
--rules blankLinesAtEndOfScope
--rules emptyBraces
--rules redundantExtensionACL
--rules redundantBreak
--rules redundantNilInit
--rules redundantObjc
--rules trailingSpace

--disable organizeDeclarations
```

**Comment by nicklockwood** [comment]
@dwirandytlvk can you share the entire source file? I can't reproduce this with just the lines shown in the screenshot, so I suspect the wrong line number is a knock-on effect from something further up in the file

**Comment by nicklockwood** [comment]
@dwirandytlvk fixed in 0.50.4
